documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,154 @@
+within Openverse. The goal is to ensure that user-facing service disruptions are
+handled promptly, efficiently, and effectively, mitigating impact on our users
+and maintaining the highest level of service availability possible.
+
+## What is an incident?
+
+An **incident** is any event which compromises or could compromise the goals of
+the Openverse project. While commonly technical problems with our services
+(Openverse.org, the Openverse API, and "official" integrations like the WP core
+integration and pattern directory integrations) these can also be incidents of a
+public relations or community nature. Here are some example incidents:
+
+- The API is serving slower-than-typical responses which occasionally time out
+- Openverse.org has a critical bug preventing search for mobile users using RTL
+  languages
+- A user has posted public comments using hateful language towards an Openverse
+  maintainer
+- Openverse.org is completely offline
+- Production resource usage is skyrocketing and could lead to downtime, but
+  there aren’t any actual issues yet
+
+All of these scenarios suggest that **Openverse behavior normally depended on or
+expected by regular users is not behaving as it should.**
+
+It is crucial to be proactive here; if something _might_ be an incident, but
+you’re not certain, it is better to record it than to ignore it. Do not assume
+"someone else" is handling an incident unless they have explicitly recorded it
+using the process defined here.
+
+## How do we _identify_ incidents?
+
+Incidents can be detected through various channels such as automated alerts,
+user reports, or through manual checks by maintainers. The Mini Support Rotation
+(MSR) runner makes these checks as part of their daily responsibilities.
+
+While the MSR runner should be monitoring our communication channels closely,
+**any online maintainer who sees something that might be an incident should
+follow the incident recording process outlined here.**
+
+## How do we _classify_ incidents?
+
+Incidents are classified in two ways:
+
+- **Severity** - How impactful the incident is
+- **Status** - The current state of the incident
+
+### Severity
+
+Incidents should be classified using one of three levels of severity. It is
+important to note that regardless of severity, working to resolve incidents
+always **takes priority over regular project work and MSR duties.** Once service
+is stabilized, any ongoing investigations, preventative measures, or
+documentation changes will be prioritized by the team alongside our other work.
+
+This severity scale is intentionally distinct from our GitHub priority scale.
+Any GitHub issues created to _stabilize_ a Severity 1 or Severity 2 incident
+should be marked as "critical" in GitHub. Other issues should be prioritized at
+your own discretion.
+
+- **Severity 1**: Critical issue affecting all users and all functionalities of
+  Openverse.
```

Comment:
What does "all functionalities of Openverse" entail? I understand there's some ambiguity here that is necessary and folks need to be comfortable making a decision on the fly, but severity 1 would only happen if, for example, AWS `us-east-1` lost all availability zones. Which, to be clear, is something we can basically do nothing about in the moment.

I guess, asked another way, can you share examples of what would qualify for severity 1? A full frontend outage would be, to me, as severe as we should ever get (within our control), but would just be severity 2 in this scale (API and its users are okay).